Adyen/Core/Actions/ThreeDS2ChallengeAction.swift

```diff
@@ -8,19 +8,24 @@ import Foundation
 
 /// Describes an action in which a 3D Secure challenge is presented to the user.
 public struct ThreeDS2ChallengeAction: Decodable {
+
+    /// The 3D Secure authorization token.
+    public let authorisationToken: String?
     
     /// The 3D Secure challenge token.
     public let token: String
     
     /// The server-generated payment data that should be submitted to the `/payments/details` endpoint.
-    public let paymentData: String
+    public let paymentData: String?
     
     /// Initializes a 3D Secure challenge action.
     ///
     /// - Parameters:
+    ///   - authorisationToken: The 3D Secure authorization token.
     ///   - token: The 3D Secure challenge token.
     ///   - paymentData: The server-generated payment data that should be submitted to the `/payments/details` endpoint.
-    public init(token: String, paymentData: String) {
+    public init(authorisationToken: String?, token: String, paymentData: String?) {
```

Comment:
This is breaking change. Better do

```suggestion
    public init(token: String, paymentData: String?,authorisationToken: String? = nil) {
```